webapp/package.json

```diff
@@ -58,7 +58,7 @@
     "sass": "1.60.0",
     "sass-loader": "10.4.1",
     "style-loader": "1.2.1",
-    "typescript": "4.0.3",
+    "typescript": "4.2.3",
```

Comment:
Is this needed for this PR? As well as the changes to `package-lock.json`?

Reply:
@mickmister All the changes in package-lock.json are because of the upgrade in typescript version. The upgrade is necessary to pass the `ci-lint` check.